specifyweb/frontend/js_src/lib/components/TreeView/index.tsx

```diff
@@ -150,49 +118,65 @@ function TreeView<SCHEMA extends AnyTree>({
-    'treeEditor',
-    treeToPref[tableName],
-    'synonymColor'
-  );
+  const [isSplit, setIsSplit] = React.useState(false);
+
+  const treeContainer = () => {
+    return rows === undefined ? null : (
+      <Tree
+        treeDefinitionItems={treeDefinitionItems}
+        tableName={tableName}
+        isEditingRanks={isEditingRanks}
+        focusPath={[focusPath, setFocusPath]}
+        rows={rows}
+        actionRow={actionRow}
+        conformation={[conformation, setConformation]}
+        getRows={getRows}
+        ranks={rankIds}
+        setFocusedRow={setFocusedRow}
+        focusRef={toolbarButtonRef}
+        searchBoxRef={searchBoxRef}
+        baseUrl={baseUrl}
+      />
+    );
+  };
 
   return rows === undefined ? null : (
     <Container.Full>
-      <header className="flex items-center gap-2 overflow-x-auto sm:flex-wrap sm:overflow-x-visible">
-        <TableIcon label name={table.name} />
-        <H2 title={treeDefinition.get('remarks') ?? undefined}>
-          {treeDefinition.get('name')}
-        </H2>
-        <EditTreeDefinition treeDefinition={treeDefinition} />
-        <TreeViewSearch<SCHEMA>
-          forwardRef={searchBoxRef}
-          tableName={tableName}
-          treeDefinitionItems={treeDefinitionItems}
-          onFocusPath={setFocusPath}
-        />
-        <Button.Small
-          aria-pressed={isEditingRanks}
-          onClick={handleToggleEditingRanks}
-        >
-          {treeText.editRanks()}
-        </Button.Small>
-        <Button.Small
-          disabled={conformation.length === 0}
-          onClick={(): void => {
-            setFocusPath([0]);
-            setConformation([]);
-          }}
-        >
-          {commonText.collapseAll()}
-        </Button.Small>
-        <span className="-ml-2 flex-1" />
+      <header
+        className={`flex items-center gap-2 overflow-x-auto sm:flex-wrap sm:overflow-x-visible`}
+      >
+        <>
+          <TableIcon label name={table.name} />
+          <H2 title={treeDefinition.get('remarks') ?? undefined}>
+            {treeDefinition.get('name')}
+          </H2>
+          <EditTreeDefinition treeDefinition={treeDefinition} />
+          <TreeViewSearch<SCHEMA>
+            forwardRef={searchBoxRef}
+            tableName={tableName}
+            treeDefinitionItems={treeDefinitionItems}
+            onFocusPath={setFocusPath}
+          />
+          <Button.Small
+            aria-pressed={isEditingRanks}
+            onClick={handleToggleEditingRanks}
+          >
+            {treeText.editRanks()}
+          </Button.Small>
+          <Button.Small
+            disabled={conformation.length === 0}
+            onClick={(): void => {
+              setFocusPath([0]);
+              setConformation([]);
+            }}
+          >
+            {commonText.collapseAll()}
+          </Button.Small>
+          <Button.Small onClick={() => setIsSplit(!isSplit)}>
+            {isSplit ? treeText.join() : treeText.split()}
```

Comment:
(optional)

rename "Split" to "Split View" to be clearer to the user?

could also add `aria-pressed={isSplit}` - if you do that, then you won't need the "Join" string (could just display "Split View" always